### PR TITLE
Upgrade the version of ui-box

### PR DESCRIPTION
**Pull request by colinlohner**
# TLDR;
Upgraded ui-box version to the latest release
- Updated ui-box from 2.1.2 -> 3.2.0
- Used the new `setClassNamePrefix` method from ui-box to prevent a breaking change
- Exposed the new configureSafeHref method from ui-box

# 🎉 Thanks for taking the time to contribute to 🌲Evergreen! 🎉

It is highly appreciated that you take the time to help improve Evergreen.
We appreciate it if you would take the time to document your Pull Request.

Sadly, if we don't receive enough information, or the Pull Request doesn't
align well with our roadmap, we might respectfully
thank you for your time, and close the issue.

## Respect earns Respect 👏

Please respect our Code of Conduct, in short:

- Using welcoming and inclusive language.
- Being respectful of differing viewpoints and experiences.
- Gracefully accepting constructive criticism.
- Focusing on what is best for the community.
- Showing empathy towards other community members.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -10920,11 +10920,16 @@ react-inspector@^2.2.2:
...
     prop-types "^15.6.1"
 
-react-is@^16.12.0, react-is@^16.8.1, react-is@^16.8.4, react-is@^16.8.6, react-is@^16.9.0:
+react-is@^16.12.0, react-is@^16.8.4, react-is@^16.8.6, react-is@^16.9.0:
   version "16.12.0"
   resolved "https://registry.yarnpkg.com/react-is/-/react-is-16.12.0.tgz#2cc0fe0fba742d97fd527c42a13bec4eeb06241c"
   integrity sha512-rPCkf/mWBtKc97aLL9/txD8DZdemK0vkA3JMLShjlJB3Pj3s+lpf1KaBzMfQrAmhMQB0n1cU/SUGgKKBCe837Q==
 
+react-is@^16.8.1:
+  version "16.13.1"
```
Comment (mshwery) [inline yarn.lock]: We can dedupe this version. `npx yarn-deduplicate && yarn install`
